### PR TITLE
[17.0][MIG] project_parent

### PR DESCRIPTION
**Pull request by RicardCForgeFlow**
Migration to 17.0

@ForgeFlow

### CODE REVIEW

File: project_parent/views/project_parent_views.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </field>
+    </record>
+    <record id="view_project_project_filter" model="ir.ui.view">
+        <field name="name">project.project.filter</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_project_filter" />
+        <field name="type">search</field>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="parent_id" />
```
Comment (GuillemCForgeFlow) [inline project_parent/views/project_parent_views.xml:66]: Perhaps a good improvement would be to add a group by for the Parent Project by default, as it is also mentioned in the `readme` files.

**Review by GuillemCForgeFlow — COMMENTED** [review]
for the rest, everything looks good 💯

**Review by AndreuOForgeFlow — APPROVED** [review]
LGTM! Functional and code review 👍🏽

**Review by CRogos — COMMENTED** [review]
Could you also cleanup the commit history?
https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

Here are some samples:
image

**Comment by CRogos** [comment]
There is a lot more which should be merged: 
image

**Review by dreispt — APPROVED** [review]

**Comment by vvanneri-ia** [comment]
can anyone publish it in odoo app?

**Comment by dreispt** [comment]
I added the OCA/project 17.0 branch to the App store, but it doesn't work properly and refuses ti publish the module.
Your best alternative is the the OCA apps index: https://odoo-community.org/shop/project-parent-4017

**Pull request merged**